proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -2305,6 +2305,10 @@
           "$type": "color",
           "$value": "{basis.color.text.text-1}"
         },
+        "text-underline-offset":{
+          "$type": "other",
+          "$value": "3px"
+        },
```

Comment:
@Marwaxhello Zou je ook nog de andere states willen overnemen, zoals dat 'ie dikker wordt bij hover?

Reply:
@Marwaxhello Zou je ook even willen kijken of het zou lukken deze tokens in het basis thema te zetten, in plaats van bij Purmerend? Het Purmerend thema extend het basis thema, dan werkt het als het goed is voor beiden.